README.md

```diff
@@ -245,124 +261,25 @@ end
 | --------------- | ----------------------------------------------- | -------- | --------------------------------------------------------------- |
 | `client_id`     | The client id from your Fortnox integration     | Yes      | `nil`                                                           |
```

Comment:
It these are only needed when you manually do requests to refetch tokens we should have them as arguments to the methods that need them instead of giving the gem permanent access to something it won't need regularly.